app/views/alaveteli_pro/subscriptions/index.html.erb

```diff
@@ -39,6 +39,18 @@
           <% end %>
         </div>
 
+        <% if @referral_coupon %>
+          <div class="settings-section">
+            <h3><%= _('Refer a friend') %></h3>
+            <p class="settings__item">
+              <%= _('Share the code {{discount_code}} with a friend to give ' \
+                    'them {{discount_terms}} on signup.',
+                    discount_code: @referral_coupon.id,
```

Comment:
I think this is fine as-is, but we could consider something like this to make the coupon code bold as we're passing in the value:

```rb
discount_code: "<strong>#{ h(@referral_coupon.id) }</strong>".html_safe,
```

(maybe by way of a helper method as that's a bit untidy)

Reply:
Ah, nice. I did mention this…

> * Ideally I'd have done some nice highlighting around the coupon code itself, but that would make the translation string more complicated and probably require some design skills. I think its fine as though, tbh.

…but didn't think to try it on that side of the translation string.

I'll give this a go.

Reply:
I did see that - probably why it was on my mind while testing -but failed to quote it when making the comment, sorry!